test/WebJobs.Script.Tests/Extensions/EnvironmentExtensionsTests.cs

```diff
@@ -273,6 +273,33 @@ public void IsManagedAppEnvironment_ReturnsExpectedResult(bool isManagedAppEnvir
             Assert.Equal(expectedValue, env.IsManagedAppEnvironment());
         }
 
+        [Theory]
+        [InlineData(false, null, null, false)]
+        [InlineData(false, "", "", false)]
+        [InlineData(false, ScriptConstants.DynamicSku, null, false)]
+        [InlineData(false, null, ScriptConstants.DynamicSku, false)]
+        [InlineData(true, null, null, false)]
+        [InlineData(true, "", "", false)]
+        [InlineData(true, ScriptConstants.FlexConsumptionSku, null, false)]
+        [InlineData(true, null, ScriptConstants.DynamicSku, true)]
+        [InlineData(true, ScriptConstants.DynamicSku, null, true)]
+        [Trait(TestTraits.Group, TestTraits.LinuxConsumptionMetricsTests)]
+        public void IsV1LinuxConsumptionOnLegion_ReturnsExpectedResult(bool isLinuxConsumptionOnLegion, string websiteSku, string websiteSkuName, bool expectedValue)
```

Comment:
After rename, fix this to IsLinuxConsumptionOnLegion_ReturnsExpectedResult

Reply:
Done.

Reply:
Looks like the name still isn't correct. The method name is IsLinuxConsumptionOnLegion, so the test should be named IsLinuxConsumptionOnLegion_ReturnsExpectedResult